modules/storages/config/locales/en.yml

```diff
@@ -166,11 +167,14 @@ en:
         one_drive: "Allow OpenProject to access Azure data using OAuth to connect OneDrive/Sharepoint."
     delete_warning:
       storage: >
-        Are you sure you want to delete this storage? This will also delete the storage from all projects where it is used.
-        Further, it will also delete all links from work packages to files that are stored in that storage.
+        Are you sure you want to delete %{file_storage}? To confirm this action please introduce the
+        storage name in the field below, this will:
+      storage_delete_result_1: "Remove all storage setups for all projects using this storage."
+      storage_delete_result_2: "Remove all links from work packages of all projects to files and folders of that storage."
+      storage_delete_result_3: "In case this storage has an automatically managed project folders, these and its files will be deleted forever."
```

Comment:
Is this true?
I think that the files and folders will be hidden after the deletion.

Reply:
@ba1ash I checked the Delete service of storagas. It iterates over all ProjectStorages and calls its Delete service, which in turn deletes the project folder. So, I guess all automatically managed project folders get deleted. At least that is how I understand the code.

Reply:
@wielinde @ba1ash If I see this correctly in `modules/storages/app/services/storages/group_folder_properties_sync_service.rb:160`, we only remove permissions from the folders in Nextcloud. We do not remove them. So the text needs to be different. You agree?

Reply:
@apfohl, I agree with text changes. We only hide folders(files) on NC side.
@wielinde yes we delete project_storages record, but my point was about `these and its files will be deleted forever.`

Reply:
I changed the text.